lib/platform/git/storage.ts

```diff
@@ -1,15 +1,33 @@
-const fs = require('fs-extra');
-const { join } = require('path');
-const path = require('path');
-const URL = require('url');
-const Git = require('simple-git/promise');
-const convertHrtime = require('convert-hrtime');
+/// <reference path="../../../node_modules/simple-git/promise.d.ts" />
```

Comment:
I'm not sure this is the best practices anymore for including typings. I think you should define them in the `tsconfig.json` instead. I don't think this is really sustainable.

Reply:
I've removed that on my ts3 branch. Is no longer required.

Reply:
removed